growl.lua

```diff
@@ -86,23 +86,15 @@ function main()
   SV:finish()
 end
 
-local function getProjectPathName()
-  local projectFileName = SV:getProject():getFileName()
-  if not projectFileName then return end
-
-  local projectName, projectDir
-  projectFileName = projectFileName:gsub("\\", "/")
-  projectDir, projectName = projectFileName:match("^(.*/)([^/]+)%.svp$")
-  if not projectDir or not projectName then error(SV:T("project dir or name not found")) end
-
-  return projectName, projectDir
-end
-
 function process()
-  local projectName, projectDir = getProjectPathName()
-  local configFileName = projectDir.."RTgrowl.txt"
+  local projectFileName = SV:getProject():getFileName()
+  local configFileName = nil
+  if projectFileName ~= "" then
+    -- projectFileName may be nil if file is not saved or running in VST

```

Comment:
I didn't know that this is happening in VST

Reply:
In VST mode there is no filename stored, unless you choose save external file.